pkg/reconciliation/reconcile_racks.go

```diff
@@ -2071,6 +2082,29 @@ func (rc *ReconciliationContext) startNode(pod *corev1.Pod, labelSeedBeforeStart
 	return false, nil
 }
 
+func (rc *ReconciliationContext) waitForPodLiveness(pod *corev1.Pod) (bool, error) {
```

Comment:
I know this is still a draft, but this is something one can't do in the reconcile process. The reconcile cannot be blocked (or do anything that's long running), but instead you would need to bail out and try again.

Reply:
Got it, will update the liveness check to fail immediately and wait for reconcile to retry.